### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Antho1426**
Add numbers to sections 2, 3 and 4.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,15 +134,15 @@ We hope you will enjoy this application and that it will help you to easily orga
 
 
 <a name="sec_2"></a>
-## Technologies
+## 2. Technologies
```
Comment (RoxaneBurri) [inline README.md:137]: Why do you add the number ?

**Review by jlstoupy98 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -134,15 +134,15 @@ We hope you will enjoy this application and that it will help you to easily orga
...
 * This software is an Android app developed with Android Studio and written in Java.
 * Google Firebase is used for the backend data management (user accounts, sports activity parameters) and for the chat.
 * The location APIs available in Google Play services are used to implement our GPS tracking feature.  
 
 
 
 <a name="sec_3"></a>
-## Requirements
+## 3. Requirements
```
Comment (RoxaneBurri) [inline README.md:145]: Same

**Review by RoxaneBurri — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Antho1426) [reply README.md:137]: To get you to talk and it worked! 😆😂

File: README.md [hunk 3]
```diff
@@ -178,10 +178,10 @@ We hope you will enjoy this application and that it will help you to easily orga
...
-## Developers
+## 4. Developers
 * Roxane Burri
 * Victor Carles
 * Anthony Guinchard
 * Jean-Luc Stoupy
 * Mark Tropin
-* Kepler Warrington-Arroyo
\ No newline at end of file
+* Kepler Warrington-Arroyo
```
Comment (RoxaneBurri) [inline README.md:187]: You forgot Kepler ?

File: README.md (hunk 2 above)
Comment (Antho1426) [reply README.md:145]: Same! 🤌

File: README.md (hunk 3 above)
Comment (Antho1426) [reply README.md:187]: I don't remember to have forgotten him ^^'

**Pull request merged**